src/jbmc/jbmc_parse_options.cpp

```diff
@@ -653,6 +653,28 @@ void jbmc_parse_optionst::process_goto_function(
     remove_returns(function);
 
     replace_java_nondet(function);
+
+    // Similar removal of java nondet statements:
+    // TODO Should really get this from java_bytecode_language somehow, but we
+    // don't have an instance of that here.
```

Comment:
Pffff... (I know this code was already there)... we really have to upgrade `optionst` so that parsing can happen on setting values.